content/en/cloud_cost_management/cost_allocation/_index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Cost Allocation
+description: Learn how to allocate cloud costs across your organization with Datadog Cloud Cost Management
+further_reading:
+  - link: "/cloud_cost_management/"
+    tag: "Documentation"
+    text: "Learn about Cloud Cost Management"
+  - link: "/cloud_cost_management/cost_allocation/container_cost_allocation"
+    tag: "Documentation"
+    text: "Container Cost Allocation"
+  - link: "/cloud_cost_management/cost_allocation/bigquery"
+    tag: "Documentation"
+    text: "BigQuery Cost Allocation"
+---
+
+## Overview
```

Comment:
I have no idea where this should go but can you add a section on bigquery labels? This is a feature bigquery provides where you can add a label to your bigquery queries and then the labels show up in CCM
It is super powerful for our users and not many know about it

It is documented here https://cloud.google.com/bigquery/docs/adding-labels